frontend/playwright.config.ts

```diff
@@ -1,11 +1,108 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
+import { defineConfig, devices } from '@playwright/test';
+
+import * as testEnv from './tests/envVars';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
 
 const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
```

Comment:
The reset-project test I just added may not actually work in parallel until I modify it to involve a newly-created project with a random code, because if Chromium and Firefox both try to reset sena-3 at the same time, they could hit a race condition that makes one of them unable to perform the reset (because the other one is halfway through the process, so the project is currently empty and our code forbids starting a project reset under those conditions). But that's a problem for later. For now, yes, the tests work fine in parallel.